src/fixes.rs

```diff
@@ -104,29 +109,21 @@ fn install_lazy_loading_patches(){
         }
     }
     
-    #[hook(offset = CSS_SET_SELECTED_CHARARACTER_UI_OFFSET)]
-    pub unsafe fn css_set_selected_chararacter_ui(
-        param_1: *const u64,
-        chara_hash_1: u64,
-        chara_hash_2: u64,
-        color: u32,
-        unk1: u32,
-        unk2: u32,
-    ){
+    pub unsafe fn load_chara_1_for_ui_chara_hash_and_num(ui_chara_hash: u64, color: u32){
         // If we have the first and fourth param in our cache, then we're in the
         // character select screen and can load the files manually
         if PARAM_1 != 0 && PARAM_4 != 0 {
             // Get the color_num for smooth loading between different CSPs
             let max_color: u32 = {
                 // Get the character database for the color num function
-                let character_database = {
+                let character_database ={
                     let databases = *((*PARAMATERS_CACHE + 0x8) as *const u64);
                     *((databases + 360) as *const u64) as u64
```

Comment:
Make a proper struct so we can expand on it as needed in the future and it is clearer (but feel free to use a array of u64 to skip all the bytes we don't use yet)

Reply:
done